PlanningPhase.md

```diff
@@ -1,38 +1,56 @@
 
--&gt; Determining the purpose is important for deciding upon the functionality that may be included in the App.
+- Which functions should the app provide and 
+- which user data is necessary for the planned functionality.  
 
-### The App's Intended and Potential Use
+Especially it is important to carefully think which user data is needed and how it will be processed. 
 
-When planning an app's design, consider in what field the app shall be used. For instance, an emergency app might be intended for covering certain geographical areas, particular periods of time or predefined cases.
+- [ ] add textbox about purpose binding?
 
-An app might be used, though, for other purposes not intended in the first place. A firewall app can give you information about other apps' illegitimate connections to the internet. Yet it might also give end users a tool secretly to record and monitor other people's activities on the internet.
+**Determining the purpose is important for deciding upon the functionality that may be included in the app, and if and how user data needs to be processed.**
 
--&gt; Might the app be used in a way harmful to other people and their privacy? Consider safeguards against illegitimate uses, e.g. by ensuring transparency when the app is running.
+*For example a weather app might let you choose and display weather information; a firewall app might let you inhibit or monitor a smartphone's connections to the internet.
+For this functions certain user data might be necessary or useful like geo-data or access to data connections. This data has implications which need to be taken into account at an early stage of development for proper integration of security measures.*
 
-## Defining the App's functionality
+### Intended and Unintended Ways to Use the App
 
-A privacy-friendly app will only include functionality that is needed to serve the purpose determined in the beginning. An app providing location based services does not need to access the device's microphone to do so.
+- [ ] is the unintended use a main issue here?
 
-Any functionality that is not strictly necessary to serve the intended purpose requires additional and explicit consent by the end-user. (Kommentar: Sollte die nicht ganz raus, bzw. sollten Zusatzfunktionen mit notwendigen Zugriffsrechten optional sein, und Funktionen / Zugriffsrechte, die nicht selbsterlärend sind erklärt werden?)
 
+When planning the application design, consider not which purpose you design the app for but also take into account that users might find alternative ways to use the app which can have an impact on other users of the device.
 
-## Defining the sort of data to be used
+*For examle a firewall app can give you information about other apps' illegitimate connections to the internet. Yet it might also give end users a tool to secretly record and monitor other users' internet activities.*
```

Comment:
Another example might be using animal/pet trackers to track your child (we discussed that at one meeting).